Lib/email/header.py

```diff
@@ -78,7 +78,7 @@ def decode_header(header):
                     for string, charset in header._chunks]
     # If no encoding, just return the header with no charset.
     if not ecre.search(header):
-        return [(header, None)]
+        return [(header.encode(), None)]
```

Comment:
Shouldn't this use the `'raw-unicode-escape'` encoding like on line 139 below? This will use utf-8 instead.

Reply:
Good point.

Per [RFC 2047 section 1](https://datatracker.ietf.org/doc/html/rfc2047#section-1):

>    Generally, **an "encoded-word" is a sequence of printable ASCII
>    characters that begins with "=?", ends with "?=", and has two "?"s in
>    between.**  It specifies a character set and an encoding method, and
>    also includes the original text encoded as graphic ASCII characters,
>    according to the rules for that encoding method.

My interpretation is that, if there is no RFC2047 encoding in use, the header _should_ consist of ASCII characters only.

This repo's history doesn't go back far enough for me to be exactly sure where `raw-unicode-escape` appeared as the fallback used by the Python standard library, for when non-ASCII characters _unexpectedly_ appear in the header. It seems like a reasonable fallback though, and I'll follow along for consistency.